### PR TITLE
feat(ansible-galaxy): add completion

### PR DESCRIPTION
**Pull request by mamercad**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

New completion

**What is the current behavior? (You can also link to an open issue here)**

There is no `ansible-galaxy` completion

**What is the new behavior (if this is a feature change)?**

There will be `ansible-galaxy` completion

**Additional info:**

`ansible-galaxy -h`

### CODE REVIEW

File: src/ansible-galaxy.ts [hunk 1]
```diff
@@ -0,0 +1,486 @@
...
+  },
+  {
+    name: ["--server", "-s"],
+    description: "The Galaxy API server URL",
+  },
+  {
+    name: ["--token", "--api-key"],
+    description:
+      "The Ansible Galaxy API key which can be found at ttps://galaxy.ansible.com/me/preferences",
+  },
```
Comment (fedeci) [inline src/ansible-galaxy.ts:14]: These accept an argument

**Comment by mamercad** [comment]
@fedeci Should be good-to-go now, let me know if you want me to rebase.

**Comment by mamercad** [comment]
I did some refactoring and rebased (notice I was missing some subcommands, etc).

**Review by fedeci — APPROVED** [review]
Thanks!

**Pull request merged**